### PR TITLE
MXLPA-7: Code refactoring

### PR DESCRIPTION
**Pull request by romannick**
Ticket: MXLPA-7

### CODE REVIEW

File: MXLPersonalArmory/MXLPersonalArmory.csproj [hunk 1]
```diff
@@ -102,9 +106,6 @@
   <ItemGroup>
     <None Include="App.config" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Models\" />
-    <Folder Include="ViewModels\" />
-  </ItemGroup>
```
Comment (baiumbg) [inline MXLPersonalArmory/MXLPersonalArmory.csproj]: Revert

File: MXLPersonalArmory/Models/Logger.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{
+    class Logger
+    {
+        public static void Log(string text)
+        {
+            MainWindow.ViewModel.LogBox += text + "\n";
+            Debug.WriteLine(text);
+        }
+    }
+}
```
Comment (baiumbg) [inline MXLPersonalArmory/Models/Logger.cs]: This shouldn't be in `Models`

File: MXLPersonalArmory/IPC/BackgroundInjector.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        [DllImport("kernel32.dll", SetLastError = true)]
+        static extern bool WriteProcessMemory(IntPtr hProcess, IntPtr lpBaseAddress, byte[] lpBuffer, uint nSize, out UIntPtr lpNumberOfBytesWritten);
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        static extern IntPtr CreateRemoteThread(IntPtr hProcess, IntPtr lpThreadAttributes, uint dwStackSize, IntPtr lpStartAddress, IntPtr lpParameter, uint dwCreationFlags, IntPtr lpThreadId);
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        static extern IntPtr CreateThread(IntPtr lpThreadAttributes, uint dwStackSize, IntPtr lpStartAddress, IntPtr lpParameter, uint dwCreationFlags, IntPtr lpThreadId);
+
+        HashSet<int> OpenProcesses = new HashSet<int>();
```
Comment (baiumbg) [inline MXLPersonalArmory/IPC/BackgroundInjector.cs:34]: `Dictionary<int, PipeClient>`

**Review by baiumbg — CHANGES_REQUESTED** [review]

File: MXLPersonalArmory/IPC/BackgroundInjector.cs (hunk 3 above)
Comment (romannick) [reply MXLPersonalArmory/IPC/BackgroundInjector.cs:34]: I don't see a use case for this right now.

File: MXLPersonalArmory/MXLPersonalArmory.csproj (hunk 1 above)
Comment (romannick) [reply MXLPersonalArmory/MXLPersonalArmory.csproj]: This change is required, because of the new folders and files.

**Review by baiumbg — APPROVED** [review]

**Pull request merged**